editor/app/(workspace)/dashboard/route.ts

```
@@ -17,6 +17,34 @@ export async function GET(req: NextRequest) {
+  const { data: state } = await supabase
+    .from("user_project_access_state")
+    .select(
+      `
+        *,
+        project:project(*, organization:organization(*)),
+        document:document(*)
+      `
+    )
+    .single();
+
+  if (state && state.project) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)